### ISSUE TITLE
WhitespaceBeforeArrayInitializer check false-positive on jagged array

### ISSUE THREAD

**Issue by mkordas** [sevntu-checkstyle/sevntu.checkstyle#386]
I consider the following code as correctly formatted:
```java
int[][] nestedIntOneLine = new int[][] {{1}};
```
but violation `Array initializer should have whitespace before` is raised by WhitespaceBeforeArrayInitializer check.

I need to add space before second brace to make the check happy, but this doesn't look good:
```java
int[][] nestedIntOneLine = new int[][] { {1}};
```

**Comment by romani** [comment]
merged. 

Scheduled to be in 1.17 for now at Nov 8th, is no new functionality merged I will release 1.16.X at that date.
